extensions/csv/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Camel Quarkus CSV",
+  "labels": [
+    "integration",
+    "camel",
+    "csv"
```

Comment:
Maybe add file as label as CSV tend to be file based?

Reply:
Also should we add transformation as label for data formats, where you transform data from one to another?